integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1048,6 +1048,12 @@ func (s *DockerSuite) TestRunDeviceSymlink(c *check.C) {
 	err = os.Symlink(tmpFile, symFile)
 	c.Assert(err, checker.IsNil)
 
+	// Create a symbolic link to /dev/zero, this time with a relative path (#22271)
+	err = os.Symlink("zero", "/dev/symzero")
+	// We need to remove this symbolic link here as it is created in /dev/, not temporary directory as above
+	defer os.Remove("/dev/symzero")
+	c.Assert(err, checker.IsNil)
```

Comment:
Thanks @tiborvass just updated the pull request. Let me know if there are any other issues.